adserver/regiontopics.py

```diff
@@ -1,320 +0,0 @@
-"""Our targeting data for topics & regions."""
```

Comment:
I did actually find this useful for linking to users to show what keywords are default in each region, as well as copy/pasting the list of country codes for doing country-based targeting. It's definitely not the best solution for either of those, but I do think having some way to link someone this information is useful. Perhaps an API that just returns this data for each of our regions/topics?

Reply:
An API seems reasonable although that's still a little challenging to "show" customers. Maybe documentation as well?

Reply:
Yea, it would be best if the docs could render via API, or at least build from the API, since the topics change over time.